docs/docs/references/users/README.md

```diff
@@ -1,5 +1,91 @@
 # 👨‍👩‍👧‍👦 Users
 
+The _users_ are the main entities of the identity service.
+We will describe the user-related concepts and details in the following.
+
 ## Profile
 
+Each user has a profile containing the user information.
+
+A user profile consists of a few sections:
+
+- [**Basic data**](#basic-data): contains basic info in pre-defined attributes, such as user id, username, email, phone number, and when the user last signed in.
+- [**Custom data**](#custom-data): contains additional info in customizable attributes, such as user-preferred color and language.
+- [**Identities**](#identities): contains the user info retrieved from social sign-in, such as Facebook, GitHub, and WeChat.
+
+### All attributes
+
+The following attributes (except `password_encrypted` and `password_encryption_method`) are visible on the user profile,
+which means you can view them using Management API.
+
+:::note The meanings of some following table columns
+
+- **Unique**: Ensures the _uniqueness_ of the values entered into an attribute of a DB table.
+- **Required**: Ensures that the values entered an attribute of a DB table can NOT be NULL.
```

Comment:
Better put it under the table and name it as a footnote.

Reply:
prefer put it under the table.